src/application/components/Layouts/SettingsModal.tsx

```diff
@@ -0,0 +1,77 @@
+import { Dispatch, Fragment, MutableRefObject, SetStateAction } from "react";
+import { Dialog, Transition } from "@headlessui/react";
+
+declare const VERSION: string;
+
+export function Modal({
```

Comment:
[nit] Since this is the `SettingsModal.tsx` file, could you name this component to match? Seeing `<Modal />` used in the layout feels like I'm working with a primitive Modal component. Naming this `SettingsModal` makes it easy to understand that we are working with a specific modal implementation.